.github/labeler.yml

```
@@ -56,4 +56,5 @@
     - any-glob-to-any-file: ".github/workflows/*.yml"
 
 "СВЯТЫЕ УГОДНИКИ":
-- "Content.Server.Database/*"
+  - changed-files:
+    - any-glob-to-any-file: "Content.Server.Database/*"
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)